ferrocene/ci/docker-images/ubuntu-20/requirements.txt

```diff
@@ -0,0 +1,39 @@
+# This file was autogenerated by uv via the following command:
```

Comment:
Hmmm... I did actually make these hardlinks. I think I need to fiddle with this more.